### ISSUE TITLE
Javascript code is not color coded when inside a script tag

### ISSUE THREAD

**Issue by a20willa** [HGustavs/LenaSYS#11326]
When javascript is in a script tag inside html, there is no color coding which makes it harder to read. Check if it would be feasible to   add this, and implement it if it's possible. 

![image](https://user-images.githubusercontent.com/81626616/161699456-91731f12-e970-4e70-b37f-26b97d51c328.png)

**Comment by HGustavs** [comment]
I agree this would be a good addition but it must not be at the expense of the understanding of the current functionality.

**Comment by a20sarle** [comment]
First step needed is to research how or if this issue can be splitted.
